study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/util/FdahpStudyDesignerConstants.java

```diff
@@ -302,7 +302,7 @@
       "This section has been marked as complete";
   public static final String QUESTIONNARIE_STUDY_ID = "questionnarieStudyId";
   public static final String QUESTIONNARIES_ERROR_MSG =
-      "One or more of the study's questionnaires for the study is scheduled for a date that has already occurred. Please correct the dates and try again.";
+      "One or more of the study's activities is scheduled for a date that has already expired. Please correct the same and try again.";
```

Comment:
typo is=are and "Please correct the same" is awkward phrasing - please have tech writer help.

Reply:
Fixed